recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "y-py" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
+  sha256: ae67b4c261277ea94d9435a0581696b24b1e191f5c5b82059147d6409974ca37
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install ./y-py -vv
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - maturin
+    - pip
+    - python
+  run:
+    - python
+    - libgcc-ng  # [linux]
+
+test:
+  imports:
+    - y_py
+
+about:
+  home: https://github.com/yjs/y-crdt/y-py
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
still needs all the licenses for the rust code that gets statically compiled in. have a look an [this example with `cargo-bundle-licenses`](https://github.com/conda-forge/staged-recipes/pull/17137/files#diff-2c13f5bc53e41b3e099536e823937e8be66a3db5d2d446adf15443a4872c77e5R23:
).